exercises/J-functions/exercise.js

```diff
@@ -1,5 +1,6 @@
 function halve(number) {
   // complete the function here
+  return number - 6;
 }
 
 var result = halve(12);
```

Comment:
Hey Advocate,

Your code only works in a condition where the argument is equal to 12, when i put halve(24) or any other number except 12 it won't meet the requirement. Consider using a different operator for your code to work in different conditions. 

Thanks